test/e2e/flowaggregator_test.go

```diff
@@ -1454,9 +1462,9 @@ func getClickHouseOutput(t *testing.T, data *TestData, srcIP, dstIP, srcPort str
 	var flowRecords []*ClickHouseFullRow
 	var queryOutput string
 
-	query := fmt.Sprintf("SELECT * FROM flows WHERE (sourceIP = '%s') AND (destinationIP = '%s')", srcIP, dstIP)
+	query := fmt.Sprintf("SELECT * FROM flows WHERE (sourceIP = '%s') AND (destinationIP = '%s') AND (octetDeltaCount != 0)", srcIP, dstIP)
```

Comment:
The purpose of moving `octetDeltaCount != 0` check here is to decrease the size of flow records and shorten the test time?

Reply:
Because I removed all the `octetDeltaCount != 0` for the following record verification, we need to filter out this kind of records from the IPFIX collector and the ClickHouse.

Reply:
Yes, I was asking what's the benefit for making this change?

Reply:
The test won't fail if there is any delay when querying the IPFIX collector Pod or the ClickHouse.